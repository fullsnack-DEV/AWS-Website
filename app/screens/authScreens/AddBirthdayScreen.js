import React, {useContext, useEffect, useState, useLayoutEffect} from 'react';
import {
  View,
  Text,
  StyleSheet,
  TouchableOpacity,
  SafeAreaView,
  Image,
} from 'react-native';

import {
  widthPercentageToDP as wp,
  heightPercentageToDP as hp,
} from 'react-native-responsive-screen';

import moment from 'moment';

import {Tooltip} from 'react-native-elements';
import LinearGradient from 'react-native-linear-gradient';
import FastImage from 'react-native-fast-image';
import DatePicker from 'react-native-date-picker';

import strings from '../../Constants/String';
import colors from '../../Constants/Colors';
import fonts from '../../Constants/Fonts';
import images from '../../Constants/ImagePath';
import AuthContext from '../../auth/context';
import ActivityLoader from '../../components/loader/ActivityLoader';

export default function AddBirthdayScreen({navigation, route}) {
  const authContext = useContext(AuthContext);

  console.log(
    'auth birthday',
    new Date(authContext?.entity?.obj?.birthday * 1000),
  );
  const [dateValue, setDateValue] = useState(
    authContext?.entity?.obj?.birthday
      ? new Date(authContext?.entity?.obj?.birthday * 1000)
      : new Date(),
  );
  const [minDateValue, setMinDateValue] = useState(new Date());
  const [maxDateValue, setMaxDateValue] = useState(new Date());

  const [loading] = useState(false);

  const onChange = (selectedDate) => {
    setDateValue(selectedDate);
  };
  useLayoutEffect(() => {
    navigation.setOptions({
      headerLeft: () => (
        <TouchableOpacity
          onPress={() => {
            navigation.pop();
          }}>
          <Image
            source={images.backArrow}
            style={{
              height: 20,
              width: 15,
              marginLeft: wp('5.33%'),
              tintColor: colors.whiteColor,
            }}
          />
        </TouchableOpacity>
      ),

      headerRight: () => (
        <Text
          style={styles.nextButtonStyle}
          onPress={() => {
            const userParams = {birthday: new Date(dateValue).getTime() / 1000};
            navigateToGenderScreen(userParams);
          }}>
          Next
        </Text>
      ),
    });
  });
  useEffect(() => {
    const mindate = new Date();
    const maxdate = new Date();
    mindate.setFullYear(mindate.getFullYear() - 13);
    maxdate.setFullYear(maxdate.getFullYear() - 123);
    // setDateValue(mindate);
    setMinDateValue(mindate);
    setMaxDateValue(maxdate);
  }, []);
  const navigateToGenderScreen = (userParams) => {
    console.log('route?.params?.signupInfo', route?.params?.signupInfo);
    console.log('userParams', userParams);

    const profileData = {
      ...route?.params?.signupInfo,
      birthday: userParams.birthday,
    };
    console.log('Profile data gender ', profileData);
    navigation.navigate('ChooseGenderScreen', {
      signupInfo: {...profileData},
    });
  };

  return (
    <LinearGradient
      colors={[colors.themeColor1, colors.themeColor3]}
      style={styles.mainContainer}>
      <ActivityLoader visible={loading} />
      <View style={{flex: 1}}>
        <FastImage
          resizeMode={'stretch'}
          style={styles.background}
          source={images.loginBg}
        />

        <Text style={styles.checkEmailText}>{strings.addBirthdayText}</Text>
        <Text style={styles.resetText}>{strings.notDisplayText}</Text>
        <Tooltip
          containerStyle={{
            left: 25,
          }}
          popover={
            <View style={{flex: 1, padding: 10}}>
              <Text
                style={{
                  color: colors.themeColor,
                  fontSize: 14,
                  fontFamily: fonts.RRegular,
                }}>
                {strings.birthdatText}
              </Text>
            </View>
          }
          backgroundColor={colors.parrotColor}
          height={hp('40%')}
          width={wp('75%')}
          overlayColor={'transparent'}
          skipAndroidStatusBar={true}>
          <Text style={styles.whyAskingText}>{strings.whyAskingText}</Text>
        </Tooltip>

        {/* Date.parse(dateValue) */}

        <View style={styles.matchFeeTxt}>
          <Text style={styles.dateText}>
            {moment(dateValue).format('MMM DD, YYYY')}
          </Text>
        </View>
        <View style={{alignItems: 'center', justifyContent: 'center'}}>
          <DatePicker
            textColor={colors.whiteColor}
            fadeToColor={'none'}
            mode={'date'}
            maximumDate={minDateValue}
            minimumDate={maxDateValue}
            testID={'startsDateDateTimePicker'}
            style={{marginTop: 15}}
            date={dateValue}
            onDateChange={onChange}
          />
        </View>
      </View>
      <SafeAreaView>
        <View
          style={{
            bottom: 16,
          }}>
          <Text style={styles.birthDateChangeNote}>
            {strings.birthDateChangeNote}
          </Text>
          {/* <TCButton
          title={strings.continueCapTitle}
          onPress={() => {
            const userParams = {birthday: new Date(dateValue).getTime() / 1000};
            updateProfile(userParams, () => {
              navigation.navigate('ChooseGenderScreen');
            });
          }}
          extraStyle={{marginTop: 50}}
        /> */}
        </View>
      </SafeAreaView>
    </LinearGradient>
  );
}
const styles = StyleSheet.create({
  background: {
    height: hp('100%'),
    position: 'absolute',
    width: wp('100%'),
  },
  checkEmailText: {
    color: colors.whiteColor,
    fontFamily: fonts.RBold,
    fontSize: 25,
    marginLeft: wp('6.6%'),
    marginTop: hp('11.39%'),
    textAlign: 'left',
  },
  dateText: {
    color: colors.themeColor,
    fontFamily: fonts.RRegular,
    fontSize: 16,
  },
  mainContainer: {
    flex: 1,
    paddingTop: 25,
  },
  matchFeeTxt: {
    height: 40,
    width: wp('81.3%'),
    alignSelf: 'center',
    justifyContent: 'center',
    marginTop: hp('9.23%'),
    fontSize: wp('3.8%'),
    color: 'black',
    backgroundColor: 'rgba(255,255,255,0.9)',
    paddingLeft: 20,
    borderRadius: 5,
    shadowColor: colors.googleColor,
    shadowOffset: {width: 0, height: 1},
    shadowOpacity: 0.5,
    shadowRadius: 1,

    elevation: 3,
  },
  resetText: {
    color: colors.whiteColor,
    fontFamily: fonts.RMedium,
    fontSize: 16,
    marginLeft: wp('6.6%'),
    marginRight: 20,
    marginTop: 5,
    textAlign: 'left',
  },
  whyAskingText: {
    color: colors.parrotColor,
    fontFamily: fonts.RMedium,
    fontSize: 14,
    marginLeft: wp('6.6%'),
    marginRight: 20,
    marginTop: hp('1.84%'),
    // marginBottom: hp('9%'),
    textAlign: 'left',
  },
  birthDateChangeNote: {
    color: colors.parrotColor,
    fontFamily: fonts.RMedium,
    fontSize: 14,
    marginRight: wp('9.33%'),
    marginLeft: wp('9.33%'),
    textAlign: 'left',
  },

  nextButtonStyle: {
    fontFamily: fonts.RMedium,
    fontSize: 16,
    marginRight: wp('4%'),
    color: colors.whiteColor,
  },
});
