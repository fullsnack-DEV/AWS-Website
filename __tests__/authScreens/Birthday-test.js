/* eslint-disable no-undef */
import React from 'react';
import {render} from '@testing-library/react-native';
import AddBirthdayScreen from '../../app/screens/authScreens/AddBirthdayScreen';

jest.mock('react-native-fs', () => ({}));

describe('Birthday Screen Test', () => {
  const navigate = jest.fn();
  const setOptions = jest.fn();
  const goBack = jest.fn();

  it('should match the snapshot- AddBirthdayScreen screen', () => {
    const rendered = render(
      <AddBirthdayScreen navigation={{goBack, navigate, setOptions}} />,
    ).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
