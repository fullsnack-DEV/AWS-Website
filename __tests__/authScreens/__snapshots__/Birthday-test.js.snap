// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Birthday Screen Test should match the snapshot- AddBirthdayScreen screen 1`] = `
<BVLinearGradient
  colors={
    Array [
      4294949633,
      4294924288,
    ]
  }
  endPoint={
    Object {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    Object {
      "x": 0.5,
      "y": 0,
    }
  }
  style={
    Object {
      "flex": 1,
      "paddingTop": 25,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {
            "overflow": "hidden",
          },
          Object {
            "height": 1334,
            "position": "absolute",
            "width": 750,
          },
        ]
      }
    >
      <FastImageView
        resizeMode="stretch"
        source={
          Object {
            "default": Object {
              "Invoicing": [Circular],
              "Members": [Circular],
              "Payment_method": [Circular],
              "Payout_method": [Circular],
              "SettingPrivacy": [Circular],
              "Transations": [Circular],
              "accountMyClubs": [Circular],
              "accountMyLeagues": [Circular],
              "accountMyRefereeing": [Circular],
              "accountMySchedule": [Circular],
              "accountMyScoreKeeping": [Circular],
              "accountMySports": [Circular],
              "accountMyTeams": [Circular],
              "accountPaymentPayout": [Circular],
              "accountSettingPrivacy": [Circular],
              "addEvent": [Circular],
              "addMessageChat": [Circular],
              "addRole": [Circular],
              "addSport": [Circular],
              "addSportList": [Circular],
              "addStatus": [Circular],
              "appleLogo": [Circular],
              "archerySport": [Circular],
              "arrowDown": [Circular],
              "arrowGT": [Circular],
              "arrowGraterthan": [Circular],
              "assistBy": [Circular],
              "assistReferee": [Circular],
              "assistsImage": [Circular],
              "assistsSelected": [Circular],
              "availableChallenge": [Circular],
              "backArrow": [Circular],
              "backetballSport": [Circular],
              "backgroudPlaceholder": [Circular],
              "backgroundGrayPlceholder": [Circular],
              "backgroundPlaceholder": [Circular],
              "badmintonLoader": [Circular],
              "bandySport": [Circular],
              "baseballLoader": [Circular],
              "baseballSport": [Circular],
              "basketballLoader": [Circular],
              "bgImage": [Circular],
              "blankRatingStar": [Circular],
              "blockedChallenge": [Circular],
              "blueGradient": [Circular],
              "blueRatingStar": [Circular],
              "calNextArrow": [Circular],
              "calPrevArrow": [Circular],
              "cancelImage": [Circular],
              "cancelWhite": [Circular],
              "certiImagePick": [Circular],
              "certificateUpload": [Circular],
              "chain": [Circular],
              "challengeSentPlane": [Circular],
              "challengeeBadge": [Circular],
              "challengeeCup": [Circular],
              "challengerBadge": [Circular],
              "challengerFlag": [Circular],
              "challengerInvitePlane": [Circular],
              "chat3Dot": [Circular],
              "chatPlus": [Circular],
              "chatSearch": [Circular],
              "check": [Circular],
              "checkEditor": [Circular],
              "checkGreen": [Circular],
              "checkGreenBG": [Circular],
              "checkRoundOrange": [Circular],
              "checkWhite": [Circular],
              "checkWhiteLanguage": [Circular],
              "choosePic": [Circular],
              "closeSearch": [Circular],
              "clubC": [Circular],
              "clubCover": [Circular],
              "clubLable": [Circular],
              "clubPatch": [Circular],
              "clubPlaceholder": [Circular],
              "clubPlaceholderSmall": [Circular],
              "clubSmallIcon": [Circular],
              "clubSqure": [Circular],
              "club_ph": [Circular],
              "commentDeleteIcon": [Circular],
              "commentImage": [Circular],
              "commentReport": [Circular],
              "commentReportIcon": [Circular],
              "createClub": [Circular],
              "createEventImage": [Circular],
              "createLeague": [Circular],
              "createTeam": [Circular],
              "crossImage": [Circular],
              "curruentTime": [Circular],
              "declineChallenge": [Circular],
              "deleteIcon": [Circular],
              "deleteRecentGoal": [Circular],
              "dotImage": [Circular],
              "dottedLine": [Circular],
              "doubleTeamCreated": [Circular],
              "downArrow": [Circular],
              "dropDownArrow": [Circular],
              "dropDownArrow2": [Circular],
              "dummyPhoto": [Circular],
              "editButton": [Circular],
              "editPencil": [Circular],
              "editSection": [Circular],
              "email": [Circular],
              "emailSendIconBG": [Circular],
              "emailSent1": [Circular],
              "emailSent2": [Circular],
              "emailSent3": [Circular],
              "emailSent4": [Circular],
              "emailSent5": [Circular],
              "entityCreatedBG": [Circular],
              "feedComment": [Circular],
              "feedLike": [Circular],
              "feedPlusIcon": [Circular],
              "feedShare": [Circular],
              "feedViewCommentButton": [Circular],
              "feedViewLikeButton": [Circular],
              "feedViewShareButton": [Circular],
              "feedViewUnLike": [Circular],
              "football2Loader": [Circular],
              "footballLoader": [Circular],
              "footballSport": [Circular],
              "galleryVideoIcon": [Circular],
              "gameDetail": [Circular],
              "gameGoal": [Circular],
              "gameIn": [Circular],
              "gameOrangePlus": [Circular],
              "gameOut": [Circular],
              "gameOwnGoal": [Circular],
              "gamePause": [Circular],
              "gamePlus": [Circular],
              "gameRC": [Circular],
              "gameRecord": [Circular],
              "gameSimple": [Circular],
              "gameStart": [Circular],
              "gameYC": [Circular],
              "gamesImage": [Circular],
              "gamesSelected": [Circular],
              "goalsImage": [Circular],
              "goalsSelected": [Circular],
              "grayListIcon": [Circular],
              "greenGradient": [Circular],
              "greenQBGroup": [Circular],
              "greenQBUser": [Circular],
              "greenRatingStar": [Circular],
              "groupIcon": [Circular],
              "groupUsers": [Circular],
              "hidePassword": [Circular],
              "homeSetting": [Circular],
              "home_gps": [Circular],
              "home_search": [Circular],
              "home_setting": [Circular],
              "horizontal3Dot": [Circular],
              "imageLoadingGIF": [Circular],
              "imageUploadingGIF": [Circular],
              "infoToolTipIcon": [Circular],
              "landscapeBottomImage": [Circular],
              "landscapeTopImage": [Circular],
              "landscapeVideoImage": [Circular],
              "leagueCover": [Circular],
              "leagueDemo": [Circular],
              "leagueL": [Circular],
              "leaguePatch": [Circular],
              "leaguePlaceholder": [Circular],
              "leagueSmallIcon": [Circular],
              "leave_chat_room": [Circular],
              "likeImage": [Circular],
              "likePlay": [Circular],
              "linkCopied": [Circular],
              "localHomeFilter": [Circular],
              "localhomeOverlay": [Circular],
              "lock": [Circular],
              "loginBg": [Circular],
              "logoutIcon": [Circular],
              "manageChallengeIcon": [Circular],
              "menu": [Circular],
              "menuClose": [Circular],
              "menuGray": [Circular],
              "menuOrange": [Circular],
              "messageCamera": [Circular],
              "messageCheckboxBorder": [Circular],
              "messageIcon": [Circular],
              "messageSearchButton": [Circular],
              "messageSearchButton2": [Circular],
              "moveFlottyButton": [Circular],
              "moveIcon": [Circular],
              "multipleImagesIcon": [Circular],
              "mute": [Circular],
              "myClubs": [Circular],
              "myLeagues": [Circular],
              "myRefereeing": [Circular],
              "mySchedule": [Circular],
              "myScoreKeeping": [Circular],
              "mySports": [Circular],
              "myTeams": [Circular],
              "navigationBack": [Circular],
              "newFeed": [Circular],
              "nextArrow": [Circular],
              "noSpecicPlayer": [Circular],
              "notificationBell": [Circular],
              "orangeCheckBox": [Circular],
              "orangeGradient": [Circular],
              "orangeGradiet": [Circular],
              "orangeLayer": [Circular],
              "orangeListIcon": [Circular],
              "orangeStar": [Circular],
              "paymentPayout": [Circular],
              "pickImage": [Circular],
              "playerIcon": [Circular],
              "playerSucessBG": [Circular],
              "plus": [Circular],
              "plusInvoice": [Circular],
              "plus_round_orange": [Circular],
              "popupUpArrow": [Circular],
              "portraitVideoImage": [Circular],
              "profileLevel": [Circular],
              "profilePlaceHolder": [Circular],
              "radioCheckGreenBG": [Circular],
              "radioCheckYellow": [Circular],
              "radioRoundOrange": [Circular],
              "radioSelect": [Circular],
              "radioSelectGreen": [Circular],
              "radioSelectYellow": [Circular],
              "radioUnSelectGreen": [Circular],
              "radioUnselect": [Circular],
              "recentMatch": [Circular],
              "refIcon": [Circular],
              "referePH": [Circular],
              "refereeIcon": [Circular],
              "refereeSucessBG": [Circular],
              "refereesInImage": [Circular],
              "registerReferee": [Circular],
              "registerScorekeeper": [Circular],
              "removeSportList": [Circular],
              "reqIcon": [Circular],
              "reqeIcon": [Circular],
              "requestIn": [Circular],
              "requestOut": [Circular],
              "requestSentImage": [Circular],
              "resetColor": [Circular],
              "roundArrow": [Circular],
              "savedIcon": [Circular],
              "scheduleGray": [Circular],
              "scheduleOrange": [Circular],
              "scheduleThreeDot": [Circular],
              "scorekeeperIcon": [Circular],
              "scorekeeperPH": [Circular],
              "searchLocation": [Circular],
              "searchUser": [Circular],
              "selectAddMessageButton": [Circular],
              "selectMessageSearchButton": [Circular],
              "sendButton": [Circular],
              "settingInvoice": [Circular],
              "settingSport": [Circular],
              "shareImage": [Circular],
              "shortCard": [Circular],
              "shortComment": [Circular],
              "shortDisLike": [Circular],
              "shortLike": [Circular],
              "shortShare": [Circular],
              "showPassword": [Circular],
              "signUpFb": [Circular],
              "signUpGoogle": [Circular],
              "smallRight": [Circular],
              "soccerBackground": [Circular],
              "soccerIcon": [Circular],
              "soccerImage": [Circular],
              "soccerLoader": [Circular],
              "swapTeam": [Circular],
              "switchAccount": [Circular],
              "tabFeed": [Circular],
              "tabSelectedFeed": [Circular],
              "tab_account": [Circular],
              "tab_account_group": [Circular],
              "tab_account_group_selected": [Circular],
              "tab_account_selected": [Circular],
              "tab_home": [Circular],
              "tab_home_selected": [Circular],
              "tab_message": [Circular],
              "tab_message_selected": [Circular],
              "tab_notification": [Circular],
              "tab_notification_selected": [Circular],
              "tab_reservation": [Circular],
              "tab_reservation_selected": [Circular],
              "tagDivider": [Circular],
              "tagGreenImage": [Circular],
              "tagImage": [Circular],
              "tc_message_top_icon": [Circular],
              "teamCover": [Circular],
              "teamGreenPH": [Circular],
              "teamPH": [Circular],
              "teamPatch": [Circular],
              "teamPlaceholder": [Circular],
              "teamPlaceholderSmall": [Circular],
              "teamSmallIcon": [Circular],
              "teamSqure": [Circular],
              "teamT": [Circular],
              "team_ph": [Circular],
              "tennisAce": [Circular],
              "tennisArrowLeft": [Circular],
              "tennisArrowRight": [Circular],
              "tennisBackground": [Circular],
              "tennisFault": [Circular],
              "tennisFootFault": [Circular],
              "tennisGeneral": [Circular],
              "tennisIcon": [Circular],
              "tennisLet": [Circular],
              "tennisScore": [Circular],
              "tennisSingleHeaderIcon": [Circular],
              "tennisUnForced": [Circular],
              "tennisWinner": [Circular],
              "themeGradientImage": [Circular],
              "threeDotIcon": [Circular],
              "thumbImage": [Circular],
              "timeKnobGray": [Circular],
              "timeKnobOrange": [Circular],
              "townsCupIcon": [Circular],
              "townsCupLogo": [Circular],
              "unCheckWhiteBorder": [Circular],
              "uncheckBox": [Circular],
              "uncheckEditor": [Circular],
              "uncheckWhite": [Circular],
              "undoImage": [Circular],
              "unlikeImage": [Circular],
              "unmute": [Circular],
              "upcomingMatch": [Circular],
              "usaImage": [Circular],
              "vertical3Dot": [Circular],
              "videoFullScreen": [Circular],
              "videoLoading": [Circular],
              "videoMuteSound": [Circular],
              "videoNormalScreen": [Circular],
              "videoPauseButton": [Circular],
              "videoPauseIcon": [Circular],
              "videoPlayBtn": [Circular],
              "videoPlayButton": [Circular],
              "videoPlayIcon": [Circular],
              "videoThumb": [Circular],
              "videoUnMuteSound": [Circular],
              "volleyballLoader": [Circular],
              "warningIcon": [Circular],
              "welcomeImage1": [Circular],
              "welcomeImage2": [Circular],
              "welcomeImage3": [Circular],
              "welcomeImage4": [Circular],
              "whiteTick": [Circular],
              "whiteTrophySmall": [Circular],
              "whiteUncheck": [Circular],
              "yellowCardImage": [Circular],
              "yellowCardSelected": [Circular],
              "yellowCheckBox": [Circular],
              "yellowDownArrow": [Circular],
              "yellowQBGroup": [Circular],
              "yellowQBUser": [Circular],
              "yellowRatingStar": [Circular],
              "youtubePlayButton": [Circular],
            },
          }
        }
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      />
    </View>
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Bold",
          "fontSize": 25,
          "marginLeft": 20,
          "marginTop": 187.5,
          "textAlign": "left",
        }
      }
    >
      Add your birthday.
    </Text>
    <Text
      style={
        Object {
          "color": "#FFFFFF",
          "fontFamily": "Roboto-Medium",
          "fontSize": 16,
          "marginLeft": 20,
          "marginRight": 20,
          "marginTop": 10,
          "textAlign": "left",
        }
      }
    >
      This won’t be displayed in your public profile.
    </Text>
    <View
      collapsable={false}
    >
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#E8FFB4",
              "fontFamily": "Roboto-Regular",
              "fontSize": 14,
              "marginLeft": 20,
              "marginRight": 20,
              "marginTop": 10,
              "textAlign": "left",
            }
          }
        >
          Why is TownsCup asking for my birthday?
        </Text>
      </View>
      <Modal
        animationType="fade"
        hardwareAccelerated={false}
        onShow={[Function]}
        transparent={true}
        visible={false}
      />
    </View>
    <View
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "rgba(255,255,255,0.9)",
          "borderRadius": 5,
          "color": "black",
          "elevation": 3,
          "fontSize": 28.5,
          "height": 40,
          "justifyContent": "center",
          "marginTop": 30,
          "paddingLeft": 20,
          "shadowColor": "#616161",
          "shadowOffset": Object {
            "height": 1,
            "width": 0,
          },
          "shadowOpacity": 0.5,
          "shadowRadius": 1,
          "width": 675,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#FF8A01",
            "fontFamily": "Roboto-Regular",
            "fontSize": 16,
          }
        }
      >
        Aug 03, 2022
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <RNDatePicker
        androidVariant="iosClone"
        cancelText="Cancel"
        confirmText="Confirm"
        date={1659498209519}
        fadeToColor="none"
        maximumDate={1249271009583}
        minimumDate={-2222021260417}
        minuteInterval={1}
        mode="date"
        onChange={[Function]}
        onDateChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "height": 216,
              "width": 310,
            },
            Object {
              "marginTop": 15,
            },
          ]
        }
        testID="birthday-signup-picker"
        textColor="#FFFFFF"
        theme="auto"
        title="Select date"
      />
    </View>
  </View>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
  >
    <View
      style={
        Object {
          "bottom": 16,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#E8FFB4",
            "fontFamily": "Roboto-Regular",
            "fontSize": 14,
            "marginLeft": 35,
            "marginRight": 35,
            "textAlign": "left",
          }
        }
      >
        You can’t change your birthday once you have completed the sign-up process.
      </Text>
    </View>
  </RCTSafeAreaView>
</BVLinearGradient>
`;
